c_src/exec.cpp

```diff
@@ -546,9 +546,10 @@ int set_nice(pid_t pid,int nice, std::string& error)
 void usage(char* progname) {
     fprintf(stderr,
         "Usage:\n"
-        "   %s [-n] [-alarm N] [-debug [Level]] [-user User]\n"
+        "   %s [-n] [-root] [-alarm N] [-debug [Level]] [-user User]\n"
         "Options:\n"
         "   -n              - Use marshaling file descriptors 3&4 instead of default 0&1.\n"
+        "   -root           - Allow running as root.\n"
```

Comment:
For clarity, maybe say "Allow running processes as root."? My understanding was that you can already run erlexec as root, if child processes are being executed as a non-root user.